rails-commenting.rb

```diff
@@ -5,27 +5,27 @@
 
 # app/controller/blog_posts_controller.rb
 
-# 1)
+# 1) The class BlogPostsController is inheriting from from ApplicationController
 class BlogPostsController < ApplicationController
   def index
-    # 2)
+    # 2) An instance variable that holds all the information in the BlogPost model. Could be called upon in the views if there was a render.
     @posts = BlogPost.all
   end
 
   def show
-    # 3)
+    # 3) An instance variable that will show the posts that matchthe id's from the model BlogPost when called upon.
```

Comment:
Totally, I think I would also want to elaborate a bit more on the param, but you got the idea!